pkg/compose/watch.go

```diff
@@ -249,9 +249,19 @@ func loadDevelopmentConfig(service types.ServiceConfig, project *types.Project)
 	if err != nil {
 		return nil, err
 	}
+	// TODO: compose-go should return the canonical path with all symlinks resolved
```

Comment:
indeed.
As we can release compose-go any time (we use to just depend on HEAD, but _some_ requested we adopt versions) would it make better sense to fix this in `loader.absPath` and by the way make this method public so we can use it in watch ?